auto-sudoedit.el

```diff
@@ -19,36 +19,28 @@
   "Argument S is tramp sudo path."
   (concat "/sudo::" s))
 
-(defun auto-sudoedit-current-path ()
-  "Current path file or dir."
-  (or (buffer-file-name) list-buffers-directory))
-
 (defun auto-sudoedit-sudoedit (s)
   "Open sudoedit.  Argument S is path."
   (interactive (auto-sudoedit-current-path))
   (find-file (auto-sudoedit-tramp-path s)))
 
-(defun auto-sudoedit-sudoedit-and-kill ()
-  "Open sudoedit and kill."
-  (interactive)
-  (let ((old-buffer-name (auto-sudoedit-current-path)))
-    (kill-this-buffer)
-    (auto-sudoedit-sudoedit old-buffer-name)))
-
-(defun auto-sudoedit ()
-  "`auto-sudoedit' hook."
-  (let ((curr-path (auto-sudoedit-current-path)))
-    (unless
-        (or
-         ;; Don't activate for tramp files
-         (tramp-tramp-file-p curr-path)
-         ;; Don't activate on sudo do not exist
-         (not (executable-find "sudo")))
-      ;; Current path may not exist; back up to the first existing parent
-      ;; and see if it's writable
-      (let ((first-existing-path (f-traverse-upwards #'f-exists? curr-path)))
-        (unless (and first-existing-path (f-writable? first-existing-path))
-          (auto-sudoedit-sudoedit-and-kill))))))
+(defun auto-sudoedit (_old_func &rest _args)
```

Comment:
`_old_func` show old function.
I think that is for example `original-function` or `orig-func` is more better.